runtime/moonbase/src/lib.rs

```diff
@@ -1018,8 +1018,6 @@ impl Contains<Call> for NormalFilter {
 				pallet_proxy::Call::kill_anonymous { .. } => false,
 				_ => true,
 			},
-			// We filter EVM calls as allowing these calls can cause potential attack vectors
-			// via precompiles (e.g. proxy precompile can erroneously allow privilege escalation)
 			// See https://github.com/PureStake/sr-moonbeam/issues/30
```

Comment:
This line is actually the important one.

I'd say we should still leave the EVM comment explaining why we filter those call, but maybe rephrase it so it doesn't contain attacks